### ISSUE TITLE
Once again, mobile is broken

### ISSUE THREAD

**Issue by valtsu41** [SimonWoodburyForget/mindustry-modding#18]
#14 again i guess.

**Comment by SimonWoodburyForget** [comment]
I cannot reproduce your issue, please explain what you mean more completely.

**Comment by valtsu41** [comment]
If I open the menu and click some section it goes to the section, but as soon as i close the menu it goes to a completely different place.

**Comment by SimonWoodburyForget** [comment]
Yes, that's what I can't reproduce. I haven't changed anything about the theme since then, so is there anything particular about your environment that changed for this to occur?

**Comment by valtsu41** [comment]
This bug has been around some time, I just reported it now.

**Comment by valtsu41** [comment]
And I am still using the same phone and browser.

**Comment by SimonWoodburyForget** [comment]
Try clearing your browser cache, the old version of the website could still be cached.

**Comment by valtsu41** [comment]
Did not work.
